tests/unit/FileUploadPreview.spec.ts

```diff
@@ -203,6 +203,29 @@ describe('FileUploadPreview component', () => {
     wrapper.destroy()
   }, 30000)
 
+  it('rejects when the second page is not an accepted size', async () => {
```

Comment:
Does the test on line 163 use a PDF with multiple pages? I think it should.

It's not clear to me from the new PDF filename that the first page is the correct size but the second page is invalid. Compare this filename with the one on line 185.

Thanks for the test!

Reply:
Yeah, it does look for all pages. Since the second page has the invalid size, if it only verifies the first page, then it would not return the error.

Reply:
Test looks good mostly I think.

Agree with Severin's comments:
1.  that we should add at least another letter sized page to pdf used on L163
2. more clear naming pdf file name in this test

Reply:
It's done. The invalid pdf file is renamed to "invalidSecondPage" and another valid page is now added to the valid pdf file "letterSize".

Reply:
Perfect, thanks!